src/BuiltinUtterances.ts

```diff
@@ -1,34 +0,0 @@
-// Holder class for utterances for builtin intents
-// This could have been done as just a JSON file, but then requires build tool to include it in the lib dir
-export class BuiltinUtterances {
```

Comment:
I meant for this to stay in virtual alexa - google will have its own builtinutterances that must be loaded when using it.